src/main/resources/assets/exdeorum/lang/zh_cn.json

```diff
@@ -0,0 +1,229 @@
+  "block.exdeorum.porcelain_crucible": "陶瓷坩埚",
+  "block.exdeorum.purple_archwood_crucible": "紫色至高木坩埚",
+  "block.exdeorum.red_archwood_crucible": "红色至高木坩埚",
+  "block.exdeorum.redwood_barrel": "红木桶",
+  "block.exdeorum.redwood_crucible": "红木坩埚",
+  "block.exdeorum.redwood_sieve": "红木筛子",
+  "block.exdeorum.skyroot_barrel": "天根木桶",
+  "block.exdeorum.skyroot_crucible": "天根木坩埚",
+  "block.exdeorum.skyroot_sieve": "天根木筛子",
+  "block.exdeorum.spruce_barrel": "云杉木桶",
+  "block.exdeorum.spruce_crucible": "云杉木坩埚",
+  "block.exdeorum.spruce_sieve": "云杉木筛子",
+  "block.exdeorum.starlit_barrel": "流萤木桶",
+  "block.exdeorum.starlit_crucible": "流萤木坩埚",
+  "block.exdeorum.starlit_sieve": "流萤木筛子",
+  "block.exdeorum.stone_barrel": "石桶",
+  "block.exdeorum.umbran_barrel": "暗影木桶",
+  "block.exdeorum.umbran_crucible": "暗影木坩埚",
+  "block.exdeorum.umbran_sieve": "暗影木筛子",
+  "block.exdeorum.unfired_porcelain_crucible": "未烧制的陶瓷坩埚",
+  "block.exdeorum.warped_barrel": "诡异木桶",
+  "block.exdeorum.warped_crucible": "诡异木坩埚",
+  "block.exdeorum.warped_sieve": "诡异木筛子",
+  "block.exdeorum.willow_barrel": "柳木桶",
+  "block.exdeorum.willow_crucible": "柳木坩埚",
+  "block.exdeorum.willow_sieve": "柳木筛子",
+  "block.exdeorum.witch_water": "巫水",
+  "exdeorum.container.mechanical_hammer": "机械锤",
+  "exdeorum.container.mechanical_sieve": "机械筛子",
+  "fluid_type.exdeorum.witch_water": "巫水",
+  "generator.exdeorum.void_world": "无中生有空岛",
+  "gui.exdeorum.category.barrel_compost": "桶:堆肥",
+  "gui.exdeorum.category.barrel_compost.volume": "堆肥:%s",
+  "gui.exdeorum.category.barrel_fluid_mixing": "桶:流体混合",
+  "gui.exdeorum.category.barrel_fluid_mixing.contents_are_consumed": "消耗",
+  "gui.exdeorum.category.barrel_mixing": "桶:混合",
+  "gui.exdeorum.category.crucible_heat_source": "坩埚热源",
+  "gui.exdeorum.category.crucible_heat_source.multiplier": "熔融速率:%sX",
+  "gui.exdeorum.category.hammer": "锤",
+  "gui.exdeorum.category.lava_crucible": "坩埚",
+  "gui.exdeorum.category.sieve": "筛子",
+  "gui.exdeorum.category.sieve.average_output": "平均输出:%s",
+  "gui.exdeorum.category.sieve.by_hand_only": "不能由机械筛子筛出",
+  "gui.exdeorum.category.sieve.chance": "几率:%s%%",
+  "gui.exdeorum.category.sieve.max_output": "最大值:%s",
+  "gui.exdeorum.category.sieve.min_output": "最小值:%s",
+  "gui.exdeorum.category.water_crucible": "坩埚产水",
+  "gui.exdeorum.energy_label": "能量",
+  "gui.exdeorum.redstone_control.always": "总是",
+  "gui.exdeorum.redstone_control.label": "标签",
+  "gui.exdeorum.redstone_control.mode": "模式:",
+  "gui.exdeorum.redstone_control.powered": "充能的",
+  "gui.exdeorum.redstone_control.unpowered": "未充能的",
+  "info.exdeorum.crimson_nylium_spores": "在下界岩上右键使用，将其转化为绯红菌岩。",
+  "info.exdeorum.grass_seeds": "在泥土上右键使用，将其转化为一个草方块。",
+  "info.exdeorum.mechanical_hammer": "机械锤是一种自动锤，需要提供FE能量。它可以在没有锤子的情况下运行，但添加任何锤子都会使速度加倍，锤子上的效率附魔会进一步提高速度。它支持三种不同模式的红石控制。无中生有不提供产生FE能量的方法，你需要另一个模组来供电。",
+  "info.exdeorum.mechanical_sieve": "机械筛子是一种自动筛子，需要提供FE能量。它支持三种不同模式的红石控制。无中生有不提供产生FE能量的方法，你需要另一个模组来供电。",
```

Comment:
I see Ex Deorum is translated to Chinese in the JEI informations for Mechanical Hammer and Mechanical Sieve. Is that intentional?

Reply:
Yes. But actually I have no idea on how to translate the mod name "Ex Deorum". I just use the widely accepted translation “无中生有” of Ex Nihilo, which means creating something out of nothing. If you think it not proper to use the same translation as Ex Nihilo, I could think of another translation or just use the English name.

Reply:
I don't mind, just was curious if mod names usually get translated into Chinese or not. The translation for Ex Nihilo comes from its meaning in Latin, which is "from nothing" when translated into English. If you wanted to translate Ex Deorum, it might be helpful to know that Ex Deorum means "from the gods" when translated from Latin to English. Do what you think players will find easiest to understand.

Reply:
Most popular mods have Chinese name.
There are many mods from Ex Nihilo, for different MC versions. All those mods are called "无中生有" in daily communication. Other mods have name "Ex Nihilo: xxx", and the translation is "无中生有：xxx". While your mod's name isn't like that, I think using the form "无中生有：xxx" will help players know that this mod provide something in Ex Nihilo. In Chinese “神” means god and "赐" means give. So I would translate the mod's name to "无中生有：神赐". Do you think it OK?

Reply:
Yes, sounds good to me.

Reply:
Changed in https://github.com/thedarkcolour/ExDeorum/pull/45/commits/1514d45b93eb7e4f409d1d93102de79e710cbb42